2016-10-22 18:15:30,923 [Thread-4:4404023296] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-10-22 21:05:33,714 [Thread-19:4347400192] [django.request:256] [ERROR]- Internal Server Error: /accounts/login/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 164, in get_response
    response = response.render()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 158, in render
    self.content = self.rendered_content
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 133, in rendered_content
    template = self._resolve_template(self.template_name)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 88, in _resolve_template
    new_template = self.resolve_template(template)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 78, in resolve_template
    return loader.select_template(template, using=self.using)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader.py", line 64, in select_template
    return engine.get_template(template_name, dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/backends/django.py", line 30, in get_template
    return Template(self.engine.get_template(template_name, dirs))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 167, in get_template
    template, origin = self.find_template(template_name, dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 141, in find_template
    source, display_name = loader(name, dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loaders/base.py", line 13, in __call__
    return self.load_template(template_name, template_dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loaders/base.py", line 23, in load_template
    template = Template(source, origin, template_name, self.engine)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 190, in __init__
    self.nodelist = engine.compile_string(template_string, origin)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 261, in compile_string
    return parser.parse()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 341, in parse
    compiled_result = compile_func(self, token)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 210, in do_extends
    nodelist = parser.parse()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 341, in parse
    compiled_result = compile_func(self, token)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/defaulttags.py", line 1159, in load
    (taglib, e))
TemplateSyntaxError: 'bootstrap' is not a valid tag library: Template library bootstrap not found, tried django.templatetags.bootstrap,django.contrib.admin.templatetags.bootstrap,django.contrib.staticfiles.templatetags.bootstrap,bootstrap3.templatetags.bootstrap,allauth.account.templatetags.bootstrap,allauth.socialaccount.templatetags.bootstrap
2016-10-22 21:07:59,720 [Thread-2:4429189120] [django.request:256] [ERROR]- Internal Server Error: /accounts/login/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 164, in get_response
    response = response.render()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 158, in render
    self.content = self.rendered_content
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 135, in rendered_content
    content = template.render(context, self._request)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/backends/django.py", line 74, in render
    return self.template.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 209, in render
    return self._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 89, in render
    output = self.filter_expression.resolve(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 674, in resolve
    new_obj = func(obj, *arg_vals)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/bootstrapform/templatetags/bootstrap.py", line 13, in bootstrap
    return render(element, markup_classes)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/bootstrapform/templatetags/bootstrap.py", line 74, in render
    for field in element.visible_fields():
AttributeError: 'str' object has no attribute 'visible_fields'
2016-10-22 21:08:45,497 [Thread-2:4435742720] [django.request:256] [ERROR]- Internal Server Error: /accounts/login/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 164, in get_response
    response = response.render()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 158, in render
    self.content = self.rendered_content
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 135, in rendered_content
    content = template.render(context, self._request)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/backends/django.py", line 74, in render
    return self.template.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 209, in render
    return self._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 89, in render
    output = self.filter_expression.resolve(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 674, in resolve
    new_obj = func(obj, *arg_vals)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/bootstrapform/templatetags/bootstrap.py", line 13, in bootstrap
    return render(element, markup_classes)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/bootstrapform/templatetags/bootstrap.py", line 74, in render
    for field in element.visible_fields():
AttributeError: 'str' object has no attribute 'visible_fields'
2016-10-22 21:10:47,917 [Thread-2:4592766976] [django.request:256] [ERROR]- Internal Server Error: /accounts/login/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 164, in get_response
    response = response.render()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 158, in render
    self.content = self.rendered_content
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 135, in rendered_content
    content = template.render(context, self._request)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/backends/django.py", line 74, in render
    return self.template.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 209, in render
    return self._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 89, in render
    output = self.filter_expression.resolve(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 674, in resolve
    new_obj = func(obj, *arg_vals)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/bootstrapform/templatetags/bootstrap.py", line 13, in bootstrap
    return render(element, markup_classes)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/bootstrapform/templatetags/bootstrap.py", line 74, in render
    for field in element.visible_fields():
AttributeError: 'SafeText' object has no attribute 'visible_fields'
2016-10-22 21:11:09,769 [Thread-2:4376760320] [django.request:256] [ERROR]- Internal Server Error: /accounts/login/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 164, in get_response
    response = response.render()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 158, in render
    self.content = self.rendered_content
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 135, in rendered_content
    content = template.render(context, self._request)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/backends/django.py", line 74, in render
    return self.template.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 209, in render
    return self._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 89, in render
    output = self.filter_expression.resolve(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 674, in resolve
    new_obj = func(obj, *arg_vals)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/bootstrapform/templatetags/bootstrap.py", line 13, in bootstrap
    return render(element, markup_classes)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/bootstrapform/templatetags/bootstrap.py", line 74, in render
    for field in element.visible_fields():
AttributeError: 'str' object has no attribute 'visible_fields'
2016-10-22 21:11:36,339 [Thread-2:4378857472] [django.request:256] [ERROR]- Internal Server Error: /accounts/login/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 164, in get_response
    response = response.render()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 158, in render
    self.content = self.rendered_content
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 135, in rendered_content
    content = template.render(context, self._request)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/backends/django.py", line 74, in render
    return self.template.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 209, in render
    return self._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 89, in render
    output = self.filter_expression.resolve(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 674, in resolve
    new_obj = func(obj, *arg_vals)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/bootstrapform/templatetags/bootstrap.py", line 13, in bootstrap
    return render(element, markup_classes)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/bootstrapform/templatetags/bootstrap.py", line 74, in render
    for field in element.visible_fields():
AttributeError: 'SafeText' object has no attribute 'visible_fields'
2016-10-22 21:13:24,173 [Thread-10:4334817280] [django.request:256] [ERROR]- Internal Server Error: /accounts/login/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 164, in get_response
    response = response.render()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 158, in render
    self.content = self.rendered_content
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 135, in rendered_content
    content = template.render(context, self._request)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/backends/django.py", line 74, in render
    return self.template.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 209, in render
    return self._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 89, in render
    output = self.filter_expression.resolve(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 674, in resolve
    new_obj = func(obj, *arg_vals)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/bootstrapform/templatetags/bootstrap.py", line 13, in bootstrap
    return render(element, markup_classes)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/bootstrapform/templatetags/bootstrap.py", line 74, in render
    for field in element.visible_fields():
AttributeError: 'str' object has no attribute 'visible_fields'
2016-10-22 21:30:37,619 [Thread-8:4498788352] [django.request:256] [ERROR]- Internal Server Error: /accounts/login/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 164, in get_response
    response = response.render()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 158, in render
    self.content = self.rendered_content
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 133, in rendered_content
    template = self._resolve_template(self.template_name)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 88, in _resolve_template
    new_template = self.resolve_template(template)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 78, in resolve_template
    return loader.select_template(template, using=self.using)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader.py", line 64, in select_template
    return engine.get_template(template_name, dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/backends/django.py", line 30, in get_template
    return Template(self.engine.get_template(template_name, dirs))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 167, in get_template
    template, origin = self.find_template(template_name, dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 141, in find_template
    source, display_name = loader(name, dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loaders/base.py", line 13, in __call__
    return self.load_template(template_name, template_dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loaders/base.py", line 23, in load_template
    template = Template(source, origin, template_name, self.engine)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 190, in __init__
    self.nodelist = engine.compile_string(template_string, origin)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 261, in compile_string
    return parser.parse()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 341, in parse
    compiled_result = compile_func(self, token)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 210, in do_extends
    nodelist = parser.parse()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 341, in parse
    compiled_result = compile_func(self, token)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/defaulttags.py", line 1159, in load
    (taglib, e))
TemplateSyntaxError: 'bootstap' is not a valid tag library: Template library bootstap not found, tried django.templatetags.bootstap,django.contrib.admin.templatetags.bootstap,django.contrib.staticfiles.templatetags.bootstap,bootstrapform.templatetags.bootstap,allauth.account.templatetags.bootstap,allauth.socialaccount.templatetags.bootstap
2016-10-22 21:33:07,682 [Thread-9:4498788352] [django.request:256] [ERROR]- Internal Server Error: /accounts/login/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 164, in get_response
    response = response.render()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 158, in render
    self.content = self.rendered_content
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 133, in rendered_content
    template = self._resolve_template(self.template_name)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 88, in _resolve_template
    new_template = self.resolve_template(template)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/response.py", line 78, in resolve_template
    return loader.select_template(template, using=self.using)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader.py", line 64, in select_template
    return engine.get_template(template_name, dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/backends/django.py", line 30, in get_template
    return Template(self.engine.get_template(template_name, dirs))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 167, in get_template
    template, origin = self.find_template(template_name, dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 141, in find_template
    source, display_name = loader(name, dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loaders/base.py", line 13, in __call__
    return self.load_template(template_name, template_dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loaders/base.py", line 23, in load_template
    template = Template(source, origin, template_name, self.engine)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 190, in __init__
    self.nodelist = engine.compile_string(template_string, origin)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 261, in compile_string
    return parser.parse()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 341, in parse
    compiled_result = compile_func(self, token)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 210, in do_extends
    nodelist = parser.parse()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 341, in parse
    compiled_result = compile_func(self, token)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/defaulttags.py", line 1159, in load
    (taglib, e))
TemplateSyntaxError: 'bootstap' is not a valid tag library: Template library bootstap not found, tried django.templatetags.bootstap,django.contrib.admin.templatetags.bootstap,django.contrib.staticfiles.templatetags.bootstap,bootstrapform.templatetags.bootstap,allauth.account.templatetags.bootstap,allauth.socialaccount.templatetags.bootstap
2016-10-23 11:42:26,764 [Thread-302:4397993984] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-10-23 11:50:47,202 [Thread-331:4397993984] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-10-23 11:50:47,462 [Thread-333:4397993984] [django.request:170] [WARNING]- Not Found: /apple-touch-icon-precomposed.png
2016-10-23 11:50:47,475 [Thread-334:4397993984] [django.request:170] [WARNING]- Not Found: /apple-touch-icon.png
2016-10-23 12:07:56,201 [Thread-62:4548726784] [django.request:98] [WARNING]- Forbidden (CSRF token missing or incorrect.): /admin/auth/user/
2016-10-23 12:52:15,037 [Thread-2:4502589440] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 11, in home
    return render_to_response('base.html', {'user': request.user})
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/shortcuts.py", line 39, in render_to_response
    content = loader.render_to_string(template_name, context, using=using)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader.py", line 99, in render_to_string
    return template.render(context, request)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/backends/django.py", line 74, in render
    return self.template.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 209, in render
    return self._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 151, in render
    template = context.template.engine.get_template(template)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 167, in get_template
    template, origin = self.find_template(template_name, dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 141, in find_template
    source, display_name = loader(name, dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loaders/base.py", line 13, in __call__
    return self.load_template(template_name, template_dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loaders/base.py", line 23, in load_template
    template = Template(source, origin, template_name, self.engine)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 190, in __init__
    self.nodelist = engine.compile_string(template_string, origin)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 261, in compile_string
    return parser.parse()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 341, in parse
    compiled_result = compile_func(self, token)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/defaulttags.py", line 1010, in do_if
    nodelist = parser.parse(('elif', 'else', 'endif'))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 341, in parse
    compiled_result = compile_func(self, token)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/defaulttags.py", line 1024, in do_if
    nodelist = parser.parse(('endif',))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 317, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 423, in compile_filter
    return FilterExpression(token, self)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 633, in __init__
    filter_func = parser.find_filter(filter_name)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 429, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
TemplateSyntaxError: Invalid filter: 'gravatar'
2016-10-23 12:57:19,566 [Thread-8:4502589440] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 11, in home
    return render_to_response('base.html', {'user': request.user})
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/shortcuts.py", line 39, in render_to_response
    content = loader.render_to_string(template_name, context, using=using)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader.py", line 99, in render_to_string
    return template.render(context, request)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/backends/django.py", line 74, in render
    return self.template.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 209, in render
    return self._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 151, in render
    template = context.template.engine.get_template(template)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 167, in get_template
    template, origin = self.find_template(template_name, dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 141, in find_template
    source, display_name = loader(name, dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loaders/base.py", line 13, in __call__
    return self.load_template(template_name, template_dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loaders/base.py", line 23, in load_template
    template = Template(source, origin, template_name, self.engine)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 190, in __init__
    self.nodelist = engine.compile_string(template_string, origin)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 261, in compile_string
    return parser.parse()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 341, in parse
    compiled_result = compile_func(self, token)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/defaulttags.py", line 1159, in load
    (taglib, e))
TemplateSyntaxError: 'gravatar_filter' is not a valid tag library: Template library gravatar_filter not found, tried django.templatetags.gravatar_filter,django.contrib.admin.templatetags.gravatar_filter,django.contrib.staticfiles.templatetags.gravatar_filter,bootstrapform.templatetags.gravatar_filter,allauth.account.templatetags.gravatar_filter,allauth.socialaccount.templatetags.gravatar_filter
2016-10-23 15:32:25,598 [Thread-14:4573892608] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 10, in home
    return render_to_response('home.html', {'user': request.user})
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/shortcuts.py", line 39, in render_to_response
    content = loader.render_to_string(template_name, context, using=using)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader.py", line 35, in get_template
    return engine.get_template(template_name, dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/backends/django.py", line 30, in get_template
    return Template(self.engine.get_template(template_name, dirs))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 167, in get_template
    template, origin = self.find_template(template_name, dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 141, in find_template
    source, display_name = loader(name, dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loaders/base.py", line 13, in __call__
    return self.load_template(template_name, template_dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loaders/base.py", line 23, in load_template
    template = Template(source, origin, template_name, self.engine)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 190, in __init__
    self.nodelist = engine.compile_string(template_string, origin)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 261, in compile_string
    return parser.parse()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 345, in parse
    self.extend_nodelist(nodelist, compiled_result, token)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 60, in extend_nodelist
    super(DebugParser, self).extend_nodelist(nodelist, node, token)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 371, in extend_nodelist
    "in the template." % node)
TemplateSyntaxError: <ExtendsNode: extends 'base.html'> must be the first tag in the template.
2016-10-23 16:36:35,087 [Thread-2:4380954624] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 12, in home
    return render_to_response('home.html', {'user': request.user, 'messages': json.dumps(the_req_messages)})
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 243, in dumps
    return _default_encoder.encode(obj)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 207, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 270, in iterencode
    return _iterencode(o, 0)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 184, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: <django.contrib.messages.storage.fallback.FallbackStorage object at 0x1044a2310> is not JSON serializable
2016-10-23 16:46:30,952 [Thread-2:4479520768] [django.request:98] [WARNING]- Forbidden (CSRF token missing or incorrect.): /accounts/login/
2016-10-23 16:56:57,877 [Thread-2:4414509056] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 12, in home
    return render_to_response('home.html', {'user': request.user, 'messages': json.dump(the_req_messages)})
TypeError: dump() takes at least 2 arguments (1 given)
2016-10-23 16:57:22,277 [Thread-2:4569698304] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 12, in home
    return render_to_response('home.html', {'user': request.user, 'messages': json.dumps(the_req_messages)})
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 243, in dumps
    return _default_encoder.encode(obj)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 207, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 270, in iterencode
    return _iterencode(o, 0)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 184, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: <django.contrib.messages.storage.fallback.FallbackStorage object at 0x110a0c390> is not JSON serializable
2016-10-23 16:59:09,065 [Thread-5:4362473472] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 12, in home
    return render_to_response('home.html', {'user': request.user, 'messages': json.dumps(list(the_req_messages))})
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 243, in dumps
    return _default_encoder.encode(obj)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 207, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 270, in iterencode
    return _iterencode(o, 0)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 184, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: <django.contrib.messages.storage.base.Message object at 0x104654810> is not JSON serializable
2016-10-23 17:03:02,115 [Thread-2:4408217600] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 13, in home
    return render_to_response('home.html', {'user': request.user, 'messages': json.dumps(message_list)})
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 243, in dumps
    return _default_encoder.encode(obj)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 207, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 270, in iterencode
    return _iterencode(o, 0)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 184, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: <django.contrib.messages.storage.base.Message object at 0x107192250> is not JSON serializable
2016-10-23 17:03:20,587 [Thread-7:4408217600] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 13, in home
    return render_to_response('home.html', {'user': request.user, 'messages': json.dumps(message_list)})
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 243, in dumps
    return _default_encoder.encode(obj)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 207, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 270, in iterencode
    return _iterencode(o, 0)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 184, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: <django.contrib.messages.storage.base.Message object at 0x1071cb650> is not JSON serializable
2016-10-23 17:09:26,827 [Thread-7:4408217600] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 13, in home
    print(len(Context(messages.get_messages(request))))
TypeError: object of type 'Context' has no len()
2016-10-23 17:11:10,203 [Thread-5:4448718848] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 14, in home
    list.append(json.dumps(message))
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 243, in dumps
    return _default_encoder.encode(obj)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 207, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 270, in iterencode
    return _iterencode(o, 0)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 184, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: <django.contrib.messages.storage.base.Message object at 0x109895810> is not JSON serializable
2016-10-23 17:18:30,164 [Thread-2:4532211712] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 13, in home
    message = JsonResponse(messages.get_messages(request))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/http/response.py", line 532, in __init__
    raise TypeError('In order to allow non-dict objects to be '
TypeError: In order to allow non-dict objects to be serialized set the safe parameter to False
2016-10-23 17:19:25,959 [Thread-2:4553314304] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 13, in home
    message = JsonResponse(messages.get_messages(request), safe=False)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/http/response.py", line 535, in __init__
    data = json.dumps(data, cls=encoder)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 250, in dumps
    sort_keys=sort_keys, **kw).encode(obj)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 207, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 270, in iterencode
    return _iterencode(o, 0)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/serializers/json.py", line 112, in default
    return super(DjangoJSONEncoder, self).default(o)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 184, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: <django.contrib.messages.storage.fallback.FallbackStorage object at 0x10fa6b390> is not JSON serializable
2016-10-23 17:23:47,493 [Thread-2:4508880896] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 13, in home
    message = JsonResponse(messages.get_messages(request), safe=False)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/http/response.py", line 535, in __init__
    data = json.dumps(data, cls=encoder)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 250, in dumps
    sort_keys=sort_keys, **kw).encode(obj)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 207, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 270, in iterencode
    return _iterencode(o, 0)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/serializers/json.py", line 112, in default
    return super(DjangoJSONEncoder, self).default(o)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 184, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: <django.contrib.messages.storage.fallback.FallbackStorage object at 0x10bf9f310> is not JSON serializable
2016-10-23 17:23:49,365 [Thread-3:4513087488] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 13, in home
    message = JsonResponse(messages.get_messages(request), safe=False)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/http/response.py", line 535, in __init__
    data = json.dumps(data, cls=encoder)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 250, in dumps
    sort_keys=sort_keys, **kw).encode(obj)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 207, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 270, in iterencode
    return _iterencode(o, 0)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/serializers/json.py", line 112, in default
    return super(DjangoJSONEncoder, self).default(o)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 184, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: <django.contrib.messages.storage.fallback.FallbackStorage object at 0x10d22f810> is not JSON serializable
2016-10-23 17:45:44,331 [Thread-6:4510978048] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 15, in home
    messages_list.append({'msg': message.message, 'tag': message.tag})
AttributeError: 'Message' object has no attribute 'tag'
2016-10-23 22:27:04,717 [Thread-6:4473229312] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 14, in my_profile
    'phone': user.userprofile_set.get().phone,
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/db/models/manager.py", line 127, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/db/models/query.py", line 334, in get
    self.model._meta.object_name
DoesNotExist: UserProfile matching query does not exist.
2016-10-23 22:28:09,368 [Thread-2:4341108736] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 14, in my_profile
    'phone': user.userprofile_set.get().phone | '',
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/db/models/manager.py", line 127, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/db/models/query.py", line 334, in get
    self.model._meta.object_name
DoesNotExist: UserProfile matching query does not exist.
2016-10-23 22:28:34,671 [Thread-2:4439674880] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 14, in my_profile
    'phone': user.userprofile_set.get().phone or '',
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/db/models/manager.py", line 127, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/db/models/query.py", line 334, in get
    self.model._meta.object_name
DoesNotExist: UserProfile matching query does not exist.
2016-10-23 22:31:33,273 [Thread-2:4462743552] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 13, in my_profile
    if user.userprofile_set.get():
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/db/models/manager.py", line 127, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/db/models/query.py", line 334, in get
    self.model._meta.object_name
DoesNotExist: UserProfile matching query does not exist.
2016-10-23 22:39:14,139 [Thread-2:4553183232] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 14, in my_profile
    phone = user.userprofile_set.get().phone
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/db/models/manager.py", line 127, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/db/models/query.py", line 334, in get
    self.model._meta.object_name
DoesNotExist: UserProfile matching query does not exist.
2016-10-23 22:45:18,358 [Thread-8:4550823936] [django.request:98] [WARNING]- Forbidden (CSRF token missing or incorrect.): /accounts/profile/
2016-10-23 22:46:11,055 [Thread-9:4550823936] [django.request:98] [WARNING]- Forbidden (CSRF token missing or incorrect.): /accounts/profile/
2016-10-23 22:46:48,988 [Thread-2:4475326464] [django.request:98] [WARNING]- Forbidden (CSRF token missing or incorrect.): /accounts/profile/
2016-10-23 22:46:53,039 [Thread-3:4479533056] [django.request:98] [WARNING]- Forbidden (CSRF token missing or incorrect.): /accounts/profile/
2016-10-23 22:47:16,027 [Thread-7:4475326464] [django.request:98] [WARNING]- Forbidden (CSRF token missing or incorrect.): /accounts/profile/
2016-10-23 23:05:29,740 [Thread-2:4479520768] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 151, in get_response
    % (callback.__module__, view_name))
ValueError: The view accounts.views.my_profile didn't return an HttpResponse object. It returned None instead.
2016-10-23 23:07:09,695 [Thread-4:4456452096] [django.request:98] [WARNING]- Forbidden (CSRF token missing or incorrect.): /accounts/profile/
2016-10-23 23:18:09,921 [Thread-4:4540338176] [django.request:170] [WARNING]- Not Found: /profile/
2016-10-23 23:22:17,710 [Thread-2:4429189120] [django.request:170] [WARNING]- Not Found: /profile/
2016-10-23 23:53:56,384 [Thread-2:4582281216] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 119, in get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 368, in resolve
    sub_match = pattern.resolve(new_path)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 240, in resolve
    return ResolverMatch(self.callback, args, kwargs, self.name)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 247, in callback
    self._callback = get_callable(self._callback_str)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/lru_cache.py", line 101, in wrapper
    result = user_function(*args, **kwds)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 106, in get_callable
    mod = import_module(mod_name)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 32
    else
       ^
SyntaxError: invalid syntax
2016-10-24 00:28:26,074 [Thread-3:4494725120] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 34, in my_profile
    return HttpResponseRedirect(reverse('account_profile', args={'message': msg}))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for 'account_profile' with arguments '('message',)' and keyword arguments '{}' not found. 1 pattern(s) tried: ['accounts/profile/$']
2016-10-24 00:53:47,622 [Thread-2:4459073536] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 15, in home
    message = add_notification(request, messages.SUCCESS, 'MESSAGE TEST')
TypeError: add_notification() takes exactly 4 arguments (3 given)
2016-10-24 00:54:11,165 [Thread-2:4490006528] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 15, in home
    message = add_notification(request, messages.SUCCESS, 'MESSAGE TEST')
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/helper.py", line 7, in add_notification
    return convert_message_to_toastr(messages.get_messages(request))
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/helper.py", line 13, in convert_message_to_toastr
    messages_list.append({'msg': m.message, 'tag': message.tags})
AttributeError: 'FallbackStorage' object has no attribute 'tags'
2016-10-24 00:54:34,479 [Thread-2:4414771200] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 15, in home
    message = add_notification(request, messages.SUCCESS, 'MESSAGE TEST')
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/helper.py", line 7, in add_notification
    return convert_message_to_toastr(messages.get_messages(request))
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/helper.py", line 13, in convert_message_to_toastr
    messages_list.append({'msg': m.message, 'tag': message.tag})
AttributeError: 'FallbackStorage' object has no attribute 'tag'
2016-10-24 00:54:54,292 [Thread-2:4435480576] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 15, in home
    message = add_notification(request, messages.SUCCESS, 'MESSAGE TEST')
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/helper.py", line 7, in add_notification
    return convert_message_to_toastr(messages.get_messages(request))
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/helper.py", line 13, in convert_message_to_toastr
    messages_list.append({'msg': m.message, 'tag': message.tags})
AttributeError: 'FallbackStorage' object has no attribute 'tags'
2016-10-24 01:40:34,864 [Thread-4:4590669824] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 20, in my_profile
    save_profile(user=user, form=form)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 51, in save_profile
    profile.description = profile['description']
TypeError: 'UserProfile' object has no attribute '__getitem__'
2016-10-24 15:17:10,698 [Thread-2:4513730560] [django.request:256] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/base/views.py", line 15, in home
    return render_to_response('home.html', {'user': request.user, 'messages': message})
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/shortcuts.py", line 39, in render_to_response
    content = loader.render_to_string(template_name, context, using=using)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader.py", line 99, in render_to_string
    return template.render(context, request)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/backends/django.py", line 74, in render
    return self.template.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 209, in render
    return self._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 159, in render
    return template.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 211, in render
    return self._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/defaulttags.py", line 329, in render
    return nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/defaulttags.py", line 507, in render
    six.reraise(*exc_info)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/defaulttags.py", line 493, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for 'account_profile' with arguments '()' and keyword arguments '{}' not found. 0 pattern(s) tried: []
2016-10-24 15:17:27,947 [Thread-2:4347400192] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 37, in my_profile
    return render_to_response('my_profile.html', {'user': request.user, 'form': form}, RequestContext(request))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/shortcuts.py", line 45, in render_to_response
    using=using)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader.py", line 116, in render_to_string
    template_name, context, context_instance, dirs, dictionary)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 221, in render_to_string
    return t.render(context_instance)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 209, in render
    return self._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 159, in render
    return template.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 211, in render
    return self._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/defaulttags.py", line 329, in render
    return nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/defaulttags.py", line 493, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 433, in _reverse_with_prefix
    self._populate()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 326, in _populate
    lookups.appendlist(pattern.callback, (bits, p_pattern, pattern.default_args))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 247, in callback
    self._callback = get_callable(self._callback_str)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/lru_cache.py", line 101, in wrapper
    result = user_function(*args, **kwds)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 127, in get_callable
    (lookup_view, mod_name))
ViewDoesNotExist: Could not import 'accounts.views.public_profile'. View does not exist in module accounts.views.
2016-10-24 15:17:30,397 [Thread-3:4347400192] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 37, in my_profile
    return render_to_response('my_profile.html', {'user': request.user, 'form': form}, RequestContext(request))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/shortcuts.py", line 45, in render_to_response
    using=using)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader.py", line 116, in render_to_string
    template_name, context, context_instance, dirs, dictionary)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 221, in render_to_string
    return t.render(context_instance)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 209, in render
    return self._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 159, in render
    return template.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 211, in render
    return self._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/defaulttags.py", line 329, in render
    return nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/defaulttags.py", line 493, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 433, in _reverse_with_prefix
    self._populate()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 326, in _populate
    lookups.appendlist(pattern.callback, (bits, p_pattern, pattern.default_args))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 247, in callback
    self._callback = get_callable(self._callback_str)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/lru_cache.py", line 101, in wrapper
    result = user_function(*args, **kwds)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 127, in get_callable
    (lookup_view, mod_name))
ViewDoesNotExist: Could not import 'accounts.views.public_profile'. View does not exist in module accounts.views.
2016-10-24 15:17:51,826 [Thread-2:4479520768] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 37, in my_profile
    return render_to_response('my_profile.html', {'user': request.user, 'form': form}, RequestContext(request))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/shortcuts.py", line 45, in render_to_response
    using=using)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader.py", line 116, in render_to_string
    template_name, context, context_instance, dirs, dictionary)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 221, in render_to_string
    return t.render(context_instance)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 209, in render
    return self._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 159, in render
    return template.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 211, in render
    return self._render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/defaulttags.py", line 329, in render
    return nodelist.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/defaulttags.py", line 507, in render
    six.reraise(*exc_info)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/defaulttags.py", line 493, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for 'account_profile' with arguments '()' and keyword arguments '{}' not found. 0 pattern(s) tried: []
2016-10-24 15:19:24,171 [Thread-27:4494737408] [django.request:170] [WARNING]- Not Found: /favicon.ico
2016-10-24 15:20:37,315 [Thread-47:4494200832] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 24, in my_profile
    return HttpResponseRedirect(reverse('public_profile'))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for 'public_profile' with arguments '()' and keyword arguments '{}' not found. 0 pattern(s) tried: []
2016-10-24 15:23:01,505 [Thread-2:4349497344] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 24, in my_profile
    return HttpResponseRedirect(reverse('/'))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for '/' with arguments '()' and keyword arguments '{}' not found. 0 pattern(s) tried: []
2016-10-24 20:51:26,872 [Thread-23:4574547968] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 25, in my_profile
    return HttpResponseRedirect('users/'+user['id']+"/")
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/functional.py", line 227, in inner
    return func(self._wrapped, *args)
TypeError: 'User' object has no attribute '__getitem__'
2016-10-24 20:57:16,977 [Thread-20:4343861248] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 25, in my_profile
    return HttpResponseRedirect('users/'+user.id+"/")
TypeError: cannot concatenate 'str' and 'int' objects
2016-10-24 20:59:11,550 [Thread-3:4339535872] [django.request:170] [WARNING]- Not Found: /accounts/profile/users/20/
2016-10-24 21:02:49,351 [Thread-2:4431286272] [django.request:98] [WARNING]- Forbidden (CSRF token missing or incorrect.): /accounts/profile/
2016-10-24 21:03:07,189 [Thread-3:4431286272] [django.request:98] [WARNING]- Forbidden (CSRF token missing or incorrect.): /accounts/profile/
2016-10-24 21:03:07,933 [Thread-4:4431286272] [django.request:98] [WARNING]- Forbidden (CSRF token missing or incorrect.): /accounts/profile/
2016-10-24 21:03:08,549 [Thread-5:4431286272] [django.request:98] [WARNING]- Forbidden (CSRF token missing or incorrect.): /accounts/profile/
2016-10-24 21:03:18,709 [Thread-8:4431286272] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 25, in my_profile
    return HttpResponseRedirect(reverse('public_profile'), kwargs={'user_id': str(user.id)})
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for 'public_profile' with arguments '()' and keyword arguments '{}' not found. 1 pattern(s) tried: ['users/(?P<user_id>[0-9]+)/$']
2016-10-24 21:06:02,108 [Thread-4:4567601152] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 25, in my_profile
    return HttpResponseRedirect(reverse('public_profile'), kwargs={'user_id': user.id})
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for 'public_profile' with arguments '()' and keyword arguments '{}' not found. 1 pattern(s) tried: ['users/(?P<user_id>\\d+)/$']
2016-10-24 21:06:25,229 [Thread-5:4567601152] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 25, in my_profile
    return HttpResponseRedirect(reverse('public_profile'), kwargs={'user_id': user.id})
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for 'public_profile' with arguments '()' and keyword arguments '{}' not found. 1 pattern(s) tried: ['users/(?P<user_id>\\d+)/$']
2016-10-24 21:14:44,518 [Thread-2:4529852416] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 25, in my_profile
    return HttpResponseRedirect(reverse('public_profile'), kwargs={'user_id': str(user.id)})
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for 'public_profile' with arguments '()' and keyword arguments '{}' not found. 1 pattern(s) tried: ['users/(?P<user_id>\\d+)/$']
2016-10-24 21:14:55,654 [Thread-5:4529852416] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 25, in my_profile
    return HttpResponseRedirect(reverse('public_profile'), kwargs={'user_id': str(user.id)})
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for 'public_profile' with arguments '()' and keyword arguments '{}' not found. 1 pattern(s) tried: ['users/(?P<user_id>\\d+)/$']
2016-10-24 21:15:39,346 [Thread-2:4552921088] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 25, in my_profile
    return HttpResponseRedirect(reverse('public_profile'), kwargs={'user_id': str(user.id)})
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for 'public_profile' with arguments '()' and keyword arguments '{}' not found. 1 pattern(s) tried: ['users/(?P<user_id> \\d+)/$']
2016-10-24 21:18:08,878 [Thread-2:4408217600] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 26, in my_profile
    return HttpResponseRedirect(reverse('public_profile'), kwargs={'user_id': str(user.id)})
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for 'public_profile' with arguments '()' and keyword arguments '{}' not found. 1 pattern(s) tried: ['users/(?P<user_id>\\d+)/$']
2016-10-24 21:18:15,583 [Thread-4:4408217600] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 26, in my_profile
    return HttpResponseRedirect(reverse('public_profile'), kwargs={'user_id': str(user.id)})
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for 'public_profile' with arguments '()' and keyword arguments '{}' not found. 1 pattern(s) tried: ['users/(?P<user_id>\\d+)/$']
2016-10-24 21:22:03,830 [Thread-2:4477947904] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 26, in my_profile
    return HttpResponseRedirect(reverse('public_profile'), kwargs={'user_id': 123})
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for 'public_profile' with arguments '()' and keyword arguments '{}' not found. 1 pattern(s) tried: ['users/(?P<user_id>\\d+)/$']
2016-10-24 21:22:07,047 [Thread-3:4477947904] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 26, in my_profile
    return HttpResponseRedirect(reverse('public_profile'), kwargs={'user_id': 123})
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for 'public_profile' with arguments '()' and keyword arguments '{}' not found. 1 pattern(s) tried: ['users/(?P<user_id>\\d+)/$']
2016-10-24 21:22:50,182 [Thread-4:4355788800] [django.request:256] [ERROR]- Internal Server Error: /users/20/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 151, in get_response
    % (callback.__module__, view_name))
ValueError: The view accounts.views.public_profile didn't return an HttpResponse object. It returned None instead.
2016-10-24 21:24:33,284 [Thread-2:4379119616] [django.request:256] [ERROR]- Internal Server Error: /users/20/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 151, in get_response
    % (callback.__module__, view_name))
ValueError: The view accounts.views.public_profile didn't return an HttpResponse object. It returned None instead.
2016-10-24 21:24:44,650 [Thread-3:4379119616] [django.request:256] [ERROR]- Internal Server Error: /users/20/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 151, in get_response
    % (callback.__module__, view_name))
ValueError: The view accounts.views.public_profile didn't return an HttpResponse object. It returned None instead.
2016-10-24 21:39:04,108 [Thread-100:4546629632] [django.request:256] [ERROR]- Internal Server Error: /accounts/profile/
Traceback (most recent call last):
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/utils/decorators.py", line 110, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/nuonuo-jtl/PycharmProjects/airbnb/accounts/views.py", line 39, in my_profile
    return render_to_response('my_profile.html', {'user': request.user, 'form': form}, RequestContext(request))
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/shortcuts.py", line 45, in render_to_response
    using=using)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader.py", line 116, in render_to_string
    template_name, context, context_instance, dirs, dictionary)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 208, in render_to_string
    t = self.get_template(template_name, dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 167, in get_template
    template, origin = self.find_template(template_name, dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 141, in find_template
    source, display_name = loader(name, dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loaders/base.py", line 13, in __call__
    return self.load_template(template_name, template_dirs)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loaders/base.py", line 23, in load_template
    template = Template(source, origin, template_name, self.engine)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 190, in __init__
    self.nodelist = engine.compile_string(template_string, origin)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/engine.py", line 261, in compile_string
    return parser.parse()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 341, in parse
    compiled_result = compile_func(self, token)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/loader_tags.py", line 210, in do_extends
    nodelist = parser.parse()
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 339, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/Users/nuonuo-jtl/.virtualenvs/airbnb/lib/python2.7/site-packages/django/template/base.py", line 395, in invalid_block_tag
    raise self.error(token, "Invalid block tag: '%s'" % command)
TemplateSyntaxError: Invalid block tag: 'static'
2016-10-24 22:23:55,647 [Thread-217:4402188288] [django.request:170] [WARNING]- Not Found: /accounts/profile/users/1/
2016-10-24 22:30:15,137 [Thread-218:4402188288] [django.request:170] [WARNING]- Not Found: /accounts/profile/users/1/
2016-10-24 22:30:18,626 [Thread-219:4402188288] [django.request:170] [WARNING]- Not Found: /accounts/profile/users/1/
